README.md

```diff
@@ -134,6 +135,40 @@ Separate each step of the task to an individual task. For example a "generate ic
 
 [↑ back to Table of Contents](#table-of-contents)
 
+## Prefix tasks to bundle them
+
+Bundle your tasks with a prefix so you can execute them all at once.
+
+### Why?
+
+* Bundling helps keeping your tasks organized.
+* They are easily executed at once.
+* Your execution command stays they same when you add tasks.
+
+### How?
+
+`package.json`:
+``` js
+/* recommended */
+scripts: {
+  "build:js": "browserify",
```

Comment:
Are these real tasks? (can Browserify work without arguments? And same for PostCSS just `-u`?) Otherwise I suggest we consistently use an example across all guidelines. Would the icon example used in "Write atomic tasks" work here?

Also `-u` would seem to an anti-pattern to me as one of the other guidelines we'd like to add is "Avoid shorthand command flags" as they are hard to understand by others not (so) familiair with a command.